inscriptions/ISic001216.xml

```diff
@@ -192,36 +197,23 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140700">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MH">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Ἐπαφρᾶ<lb n="2"/>Ἱρτιανοῦ<lb n="3"/>ἀπελευ-
-                    <lb n="4"/>
-                    <supplied reason="lost">θέρου</supplied>.
-                    <lb n="5"/>
-                </ab>
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI334500">
-                <ab>
-                    <lb n="1"/>
-
-
-                    Ἐπαφρᾶ<lb n="2"/>Ἱρτιανο
-                    <unclear>ῦ</unclear>
-                    <lb n="3"/>ἀπελευ-
-                    <lb n="4"/>
-                    <supplied reason="lost">θέρ</supplied>
-                    <unclear>ου</unclear>.
-                    <lb n="5"/>
+                    <lb n="1"/><name>Ἐπαφρᾶ</name>
+                    <lb n="2"/><name>Ἱρτιανοῦ</name>
+                    <lb n="3"/><unclear>ἀπ</unclear>ελευ
+                    <lb n="4" break="no"/><supplied reason="lost">θέρου</supplied>.
```

Comment:
could you alter this, to follow the better more recent text of I.Lipara, which you can find easily at:
https://epigraphy.packhum.org/text/334500